doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

```diff
@@ -258,3 +258,12 @@ Configure radio transmitter power
    :end-before: radio_power_end
 
 See :ref:`ug_matter_gs_transmission_power` for more information.
+
+Disabling unused RAM sections
+*****************************
```

Comment:
```suggestion
Disable unused RAM sections
***************************
```
Since the other section header on this page follow the present tense not continuous (just for consistency).